spec/helpers/hiring_staff/job_creation_helper_spec.rb

```diff
@@ -136,6 +136,8 @@
   end
 
   describe '#set_visited_step_class' do
+    let(:vacancy) { double('vacancy') }
```

Comment:
Does `instance_double` work here? If so, it's slightly better practice as it will give you warning if the test calls a method that doesn't exist for real `Vacancy`s.